docs/tutorials/extending_admin_ui/3_creating_a_content_list.md

```diff
@@ -0,0 +1,209 @@
+            new Criterion\Visibility(Criterion\Visibility::VISIBLE),
+        ];
+
+        $query->query = new Criterion\LogicalAnd($criterions);
+
+        $paginator = new Pagerfanta(
+            new LocationSearchAdapter($query, $this->searchService)
+        );
+        $paginator->setMaxPerPage(8);
+        $paginator->setCurrentPage($page);
+
+        return $this->render('@EzSystemsExtendingTutorial/list/all_content_list.html.twig', [
+            'totalCount' => $paginator->getNbResults(),
+            'articles' => $paginator,
+        ]);
+    }
+}
+```
+
+The highlighted line 41 indicates the template that will be used to display the list.
+
+## Add a template
+
+Finally, create an `all_content_list.html.twig` file in `src/EzSystems/ExtendingTutorialBundle/Resources/views/list`:
+
+``` twig
+{% extends 'EzPlatformAdminUiBundle::layout.html.twig' %}
+
+{% block title %}{{ 'Content List'|trans }}{% endblock %}
+
+{%- block breadcrumbs -%}
+    {% include '@EzPlatformAdminUi/parts/breadcrumbs.html.twig' with { items: [
+        { value: 'url.list'|trans|desc('Content List') }
+    ]} %}
+{%- endblock -%}
+
+{%- block page_title -%}
+    {% include '@EzPlatformAdminUi/parts/page_title.html.twig' with {
+        title: 'url.list'|trans|desc('Content List'),
+        iconName: 'article'
+    } %}
+{%- endblock -%}
+
+{%- block content -%}
+    <section class="container my-4">
+        <div class="ez-table-header">
+            <div class="ez-table-header__headline">{{ "Content List"|trans }}</div>
+        </div>
+        <table class="table">
+            <thead>
+                <tr>
+                    <th>{{ 'Content name'|trans }}</th>
+                    <th>{{ 'Content Type'|trans }}</th>
+                    <th>{{ 'Modified'|trans }}</th>
+                    <th>{{ 'Published'|trans }}</th>
+                </tr>
+            </thead>
+            <tbody>
+            {% for article in articles %}
+                <tr>
+                    <td><a href={{path('ez_urlalias', {'contentId': article.contentInfo.id})}}>{{ ez_content_name(article.contentInfo) }}</a></td>
+                    <td>{{ article.contentInfo.contentTypeId }}</td>
+                    <td>{{ article.contentInfo.modificationDate|localizeddate( 'short', 'medium' ) }}</td>
+                    <td>{{ article.contentInfo.publishedDate|localizeddate( 'short', 'medium' ) }}</td>
+                </tr>
+            {% endfor %}
+            </tbody>
+        </table>
+        {{ pagerfanta(articles, 'ez') }}
+    </section>
+{%- endblock -%}
+```
+
+## Check results
+
+!!! tip
+
+    If you cannot see the results, clear the cache and reload the application.
+
+At this point you can go to the Back Office and under "Content" you will see the new "Content list" item.
+Select it and you will see the list of all Content items in the Repository.
+
+![Content list with unfiltered results](img/content_list_unfiltered.png "Content list with unfiltered results")
```

Comment:
Works fine, but I've got the title like this: 
![screen shot 2018-04-03 at 13 35 14](https://user-images.githubusercontent.com/32538832/38247116-62e9e068-3744-11e8-9f04-f661398531a5.png)
Maybe something is missing in configuration?

Reply:
This is weird. The name is set here: https://github.com/ezsystems/developer-documentation/pull/206/files#diff-efbcd8677f994707ace9abe759407e1dR166, I've just checked and I can simply change it there in the template, no more config is needed for me...

Reply:
Fun fact: it works in `dev` mode for me, but doesn't in `prod`. Which mode do you use? :)